packages/forms/src/UIForm/fields/CheckBox/SimpleCheckBox.component.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control, jsx-a11y/no-autofocus */
```

Comment:
`jsx-a11y/no-autofocus`: In widgets, autofocus is set on the input, depending on the autofocus props. We let the control to the user, this is equivalent to a prop drilling.
`jsx-a11y/label-has-associated-control`: In widgets like Checkbox, we have a `<label>` markup that wrap the label and the input. This is supported in a11y, just that this partiicular markup is not supported by the linter rule